src/Tizen.NUI.Components/Controls/Button.cs

```diff
@@ -277,55 +243,117 @@ public enum IconOrientation
         }
 
         /// <summary>
-        /// Style of the button.
+        /// Button's icon part.
         /// </summary>
-        /// <since_tizen> 8 </since_tizen>
-        public new ButtonStyle Style => ViewStyle as ButtonStyle;
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ImageView ButtonIcon
+        {
+            get
+            {
+                if (null == buttonIcon)
+                {
+                    buttonIcon = new ImageView();
+                    Add(buttonIcon);
+                }
+                return buttonIcon;
+            }
+            set
```

Comment:
How about changing setter to an internal API?

Reply:
Or setter may need to reset like below,
```
Button.viewStyle.Icon = value.Style;
```

Reply:
Fixed